lib/board.rb

```diff
@@ -64,19 +64,20 @@ def place(ship, coordinates)
     end
 
     def render(player_board = false)
-        # require 'pry'; binding.pry
-        board  = "  1 2 3 4 \n" +
+
+        if player_board == false
+            board  = "  1 2 3 4 \n" +
                  "A #{@cells["A1"].render} #{@cells["A2"].render} #{@cells["A3"].render} #{@cells["A4"].render}\n" +
-                 "B . . . . \n" +
-                 "C . . . . \n" +
-                 "D . . . . \n"
-        
+                 "B #{@cells["B1"].render} #{@cells["B2"].render} #{@cells["B3"].render} #{@cells["B4"].render}\n" +
+                 "C #{@cells["C1"].render} #{@cells["C2"].render} #{@cells["C3"].render} #{@cells["C4"].render}\n" +
+                 "D #{@cells["D1"].render} #{@cells["D2"].render} #{@cells["D3"].render} #{@cells["D4"].render}\n"
+        else
+            board  = "  1 2 3 4 \n" +
+                 "A #{@cells["A1"].render(true)} #{@cells["A2"].render(true)} #{@cells["A3"].render(true)} #{@cells["A4"].render(true)}\n" +
```

Comment:
Let's see if there is a way (enumerables?) to make this string interpolated code shorter and more dynamic.

Reply:
I like that idea!